src/Microsoft.IdentityModel.Tokens/SecurityTokenDescriptor.cs

```diff
@@ -40,6 +41,11 @@ public class SecurityTokenDescriptor
         /// </summary>
         public string Audience { get; set; }
 
+        /// <summary>
+        /// Defines the compression algorithm that will be used to compress the JWT token payload.
```

Comment:
indenting...

Reply:
I can't see any indenting issues in Visual Studio...